### PR TITLE
Panic: Add Handler

### PR DESCRIPTION
**Pull request by xiekeyang**
@stevvooe 
Enabling log hook to capture the registry panic, when it happens.
Refer to #394.

Signed-off-by: xiekeyang <xiekeyang@huawei.com>

### CODE REVIEW

File: cmd/registry/main.go [hunk 1]
```diff
@@ -273,3 +274,14 @@ func debugServer(addr string) {
 		log.Fatalf("error listening on debug interface: %v", err)
 	}
 }
+
+func panicHandler(handler http.Handler) http.Handler {
```
Comment (stevvooe) [inline cmd/registry/main.go]: Could you please document this function.

**Comment by xiekeyang** [comment]
@stevvooe  Done.

**Comment by stevvooe** [comment]
LGTM!

@RichardScothern @dmcgowan @endophage

**Comment by dmp42** [comment]
LGTM

**Pull request merged**